src/uproot_browser/plot_view.py

```diff
@@ -54,6 +54,17 @@ def __rich_console__(
         try:
             canvas = make_plot(self.item, width, height)
             yield rich.text.Text.from_ansi(canvas)
+        except uproot_browser.plot.EmptyTreeError:
+            yield rich.panel.Panel(
+                rich.align.Align.center(
+                    rich.pretty.Pretty(f"{self.item.name} is EMPTY", no_wrap=True),
```

Comment:
```suggestion
                    f"[green]{self.item.name} is EMPTY",
```

Wouldn't this be better? You don't need quotes around the string.

Reply:
SGTM. :+1: I'll be honest here in that I did the minimal amount of reading about `rich`'s API as I haven't actually used it in a project of my own yet (:disappointed:).